README.md

```diff
@@ -150,7 +150,7 @@ A value of `max_rolledover_indices=20` on a three node bare-metal cluster with t
 
 ends up consuming a constant of `407GiB` per node.
 
-The following is an example of parameters that could be sent to the challenge.
+The following is an example configurable parameters for this challenge.
```

Comment:
I think `of` is missing? i.e. should be:

> The following is an example of configurable parameters for this challenge.

instead of

> The following is an example configurable parameters for this challenge.

Reply:
Good catch fixed